column_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2012 The Go Authors. All rights reserved.
+
+// Use of this source code is governed by a BSD-style
+
+// license that can be found in the LICENSE file.
+
+package odbc
+
+import (
+	"fmt"
+	"testing"
+	"unsafe"
+)
+
+func TestBufferLen_IsNull(t *testing.T) {
```

Comment:
I can see that you added test as I requested here https://github.com/alexbrainman/odbc/issues/182#issuecomment-2322727488 , but I was asking for a different test. Your test does not prove that there is a problem with current code.

Perhaps if you adjust `TestMSSQLTypes` https://github.com/alexbrainman/odbc/blob/bcbcb6842ce91a3e09a88335980f7f01f72a991c/mssql_test.go#L704 or `TestMSSQLTextColumnParamTypes` https://github.com/alexbrainman/odbc/blob/bcbcb6842ce91a3e09a88335980f7f01f72a991c/mssql_test.go#L1271 or any other existing test, then I can be convinced.

Thank you.

Reply:
I have added test

Reply:
This issue does not occur in all databases, only some databases exist. I think setting the size to p.size is the correct approach

Reply:
> This issue does not occur in all databases, only some databases exist.

If you have a way for me to reproduce the issue, I will look at it again.

> I think setting the size to p.size is the correct approach

I disagree.